test/integration/backward_compatible/parallel/serialization/compact/CompactSerializersLiveTest.js

```diff
@@ -0,0 +1,194 @@
+        });
+
+        it('should read server side data', async function() {
+            const expectedAge = 23;
+            const expectedId = 456;
+
+            const script = `
+                var EmployeeDTO = Java.type('example.serialization.EmployeeDTO');
+                var map = instance_0.getMap("${mapName}");
+                map.set(1.0, new EmployeeDTO(${expectedAge}, ${expectedId}));
+            `;
+            await RC.executeOnController(cluster.id, script, Lang.JAVASCRIPT);
+            const map = await client.getMap(mapName);
+            const value = await map.get(1);
+            value.should.be.instanceof(compactUtil.EmployeeDTO);
+            value.age.should.be.equal(expectedAge);
+            value.id.equals(expectedId).should.be.true;
+        });
+
+        it('should write correct data', async function() {
+            const expectedAge = 23;
+            const expectedId = Long.fromNumber(456);
+
+            const map = await client.getMap(mapName);
+            await map.set(1, new compactUtil.EmployeeDTO(expectedAge, expectedId));
+
+            const script = `
+                var map = instance_0.getMap("${mapName}");
+                var value = map.get(1.0);
+                result = value.class.toString() + value.age.toString() + value.id.toString();
+            `;
+            const response = await RC.executeOnController(cluster.id, script, Lang.JAVASCRIPT);
+
+            const resultString = response.result.toString();
+            // due to class loader, server gets the class object itself not generic record
+            resultString.should.be.eq('class example.serialization.EmployeeDTO23456');
+        });
+    });
+
+    describe('CompactSerializers with Server Explicit Config Test', function () {
+        let cluster;
+        let client;
+        let mapName;
+
+        const COMPACT_ENABLED_WITH_SERIALIZER_XML = `
+            <hazelcast xmlns="http://www.hazelcast.com/schema/config"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://www.hazelcast.com/schema/config
+                http://www.hazelcast.com/schema/config/hazelcast-config-5.0.xsd">
+                <network>
+                    <port>0</port>
+                </network>
+                <serialization>
+                    <compact-serialization enabled="true">
+                         <registered-classes>
+                            <class
+                             type-name="example.serialization.EmployeeDTO"
+                             serializer="example.serialization.EmployeeDTOSerializer"
+                             >
+                                example.serialization.EmployeeDTO
+                            </class>
+                    </registered-classes>
+                    </compact-serialization>
+                </serialization>
+            </hazelcast>
+        `;
+
+        before(async function () {
```

Comment:
these two tests look almost identical to me. can we do something to get rid of duplication?